seacrowd/sea_datasets/alorese/alorese.py

```diff
@@ -60,40 +62,23 @@
 
 
 class AloreseDataset(datasets.GeneratorBasedBuilder):
-    """Alorese Corpus is a collection of language data in a couple of Alorese variation (Alor and Pantar Alorese). The collection is available in video, audio, and text formats with genres ranging from Experiment or task, Stimuli, Discourse, and Written materials."""
+    """Alorese Corpus is a collection of language data in a couple of Alorese variation (Alor and Pantar Alorese). The collection is available in video, audio, and text formats with genres ranging
+    from Experiment or task, Stimuli, Discourse, and Written materials."""
 
-    SUBSETS = ["t2t", "sptext", "sptext_trans"]
-
-    BUILDER_CONFIGS = [
-        SEACrowdConfig(
-            name=f"{_DATASETNAME}_{subset}_source",
-            version=datasets.Version(_SOURCE_VERSION),
-            description=f"{_DATASETNAME} source schema for {subset} subset",
-            schema="source",
-            subset_id=f"{_DATASETNAME}_{subset}",
-        )
-        for subset in SUBSETS
-    ] + [
+    BUILDER_CONFIGS = [SEACrowdConfig(name=f"{_DATASETNAME}_source", version=datasets.Version(_SOURCE_VERSION), description=f"{_DATASETNAME} source schema", schema="source", subset_id=f"{_DATASETNAME}",)] + [
```

Comment:
can we fix this formatting? :D

Reply:
bueno @sabilmakbar ! it's done, sorry I forgot to delete the old `]` bracket